scripts/after_install.sh

```diff
@@ -30,7 +30,11 @@ sudo -H -u datamade $VENV_DIR/bin/pip install --upgrade setuptools
 
 # Install the project requirements into the deployment specific virtual
 # environment.
-sudo -H -u datamade $VENV_DIR/bin/pip install -r $PROJECT_DIR/requirements.txt --upgrade
+sudo -H -u datamade $VENV_DIR/bin/pip install -r $PROJECT_DIR/requirements.txt -r $PROJECT_DIR/dev-requirements.txt --upgrade
+
+# Install NPM dependencies.
+$VENV_DIR/bin/docker-compose build
+$VENV_DIR/bin/docker-compose run --rm frontend build
```

Comment:
Note that we'll have to make sure Docker is installed on the servers before this will run properly.

Reply:
We plan to spin up a designated server for just-spaces production (when the time comes): it can certainly have docker. 

The staging site has `Docker version 18.09.1, build 4c52b90`.